modules/core/src/test/java/org/apache/synapse/config/xml/endpoints/DefaultEndpointSerializerTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.synapse.config.xml.endpoints;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.synapse.endpoints.DefaultEndpoint;
+import org.apache.synapse.config.xml.AbstractTestCase;
+
+/**
+ * Test class for building and then serializing default endpoint
+ */
+public class DefaultEndpointSerializerTest extends AbstractTestCase {
+    public void test() throws Exception {
```

Comment:
Looking at a glance, I felt something wrong with the test case because `@Test` annotation was missing. Could you add a small comment on `test()` method explain the behavior?

Reply:
Eventhough @test annotation is not mandatory. Please add that for readability purposes.

Reply:
Fixed with https://github.com/wso2/wso2-synapse/pull/1105